alacritty_terminal/src/tty/unix.rs

```diff
@@ -143,7 +143,9 @@ fn default_shell_command(pw: &Passwd<'_>) -> Command {
     // -f: Bypasses authentication for the already-logged-in user.
     // -l: Skips changing directory to $HOME and prepending '-' to argv[0].
     // -p: Preserves the environment.
-    login_command.args(["-flp", pw.name, "/bin/sh", "-c", &exec]);
+    //
+    // XXX: we use bash here over sh due to `exec -a`.
+    login_command.args(["-flp", pw.name, "/bin/bash", "-c", &exec]);
```

Comment:
Why did you decide to go for bash over zsh?

Reply:
Idk, that's what I concluded. Could use zsh.

Reply:
Honestly just use zsh, has been supported forever according to @chrisnc so should be fine.